addon/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "tracked-toolbox",
+  "version": "1.2.3",
+  "description": "Helpful autotracking utilities",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:tracked-tools/tracked-toolbox.git"
+  },
+  "license": "MIT",
+  "author": "",
+  "exports": {
+    ".": "./dist/index.js",
+    "./addon-main.js": "./addon-main.js"
+  },
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "files": [
+    "addon-main.js",
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "lint": "eslint . --cache",
+    "lint:fix": "eslint . --fix",
+    "prepublishOnly": "cp ../README.md ../LICENSE.md . && rollup --config",
```

Comment:
there was discussion in https://github.com/embroider-build/embroider/pull/1188 and seem it's better DX to keep those *.md files in the root and copy at publish time + add them to root `.gitignore`